backend/api/views.py

```diff
@@ -118,20 +121,33 @@ def access_and_submission_check(user, data):
 
         # Create all contact Access objects
         Access.objects.create(sac=sac, role="editor", email=user.email, user=user)
-        Access.objects.create(
+        cert_auditee_access = Access.objects.create(
```

Comment:
@timoballard I added these here because in trying to test this PR I ran into an issue that we discussed way back: if someone is logged into the system when they're added to a submission, they won't be able to access that submission until they log out and in again. Leaving that in place makes testing this a lot harder, since it would mean that you'd have to log out & in between steps (or have multiple users involved).

Let me know if you think this is reasonable enough to leave as part of this PR or if you think we should make a separate PR for it.

Reply:
Also I just learned that if you select a bunch of lines and then add a comment, it just defaults to the first line… most of what I'm referring to here is below, in the `to_reify` loop.

Reply:
removed as this was implemented separately in #1778